.github/workflows/wheels.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 env:
   # Python 3.12+ is handled using the stable ABI, no need to build later versions
-  CIBW_PROJECT_REQUIRES_PYTHON: ">=3.8, <=3.12"
+  CIBW_PROJECT_REQUIRES_PYTHON: ">=3.8, <3.13"
```

Comment:
This isn't needed, cibuildwheel will not rebuild but will only test.

Reply:
When I tried earlier today, the build matrix initially didn't include v3.12 at all. I made this change thinking that maybe 3.12b1 doesn't satisfy the <=3.12 test. But a few things were changing at the same time, and it's possible that it was related to that.

Are you saying that the change from `<=3.12` to `<3.13` isn't needed, or that this entire line should go?

Reply:
It should be okay to be completely removed.

Reply:
Also, most of this should be set in the pyproject.toml, not as environment variables, so that you can build this locally too. Or on other CI providers.

Reply:
```suggestion
```